core/domain/storage_model_audit_jobs_test.py

```diff
@@ -59,17 +59,16 @@ def test_all_models_have_audit_jobs(self):
 
         # As models.NAMES is an enum, it cannot be iterated. So we use the
         # __dict__ property which can be iterated.
-        for name in models.NAMES.__dict__:
-            if '__' not in name:
-                all_model_module_names.append(name)
+        for name in models.NAMES:
+            all_model_module_names.append(name)
 
         names_of_ndb_model_subclasses = []
         for module_name in all_model_module_names:
             # We skip base models since there are no specific audit jobs
             # for base models. The audit jobs for subclasses of base models
             # cover the test cases for base models, so extra audit jobs
             # for base models are not required.
-            if module_name == 'base_model':
+            if module_name.value == 'base_model':
```

Comment:
Can you do `module_name == models.NAMES.base_model` here?

Reply:
Yes. Done.